charts/agent/templates/securitycontextconstraint.yaml

```diff
@@ -15,7 +15,8 @@ allowHostPID: true
 allowHostPorts: false
 allowPrivilegeEscalation: true
 allowPrivilegedContainer: true
-allowedCapabilities: []
+allowedCapabilities:
+- '*'
```

Comment:
I suppose we should use `{{- if eq "true" (include "agent.privileged" .) }}` as well and set the proper capabilities when needed instead of adding `*`